README.md

```diff
@@ -1,4 +1,6 @@
-# Karton Playground: getting started
+# Karton Playground
+
+![](./img/logo-sm.png)
```

Comment:
Maybe use `width` property instead of uploading multiple files to the repo? :thinking:

Reply:
example: 
`<img src="https://raw.githubusercontent.com/CERT-Polska/karton-playground/020f031820b99ab9f12841266c0043fa3fcf69bc/img/logo.png" width=50>`

<img src="https://raw.githubusercontent.com/CERT-Polska/karton-playground/020f031820b99ab9f12841266c0043fa3fcf69bc/img/logo.png" width=50>

Reply:
I don't even know if we want the big or small one :thinking: but right

Reply:
Changed to full-size logo (I don't know how big it should be :thinking: )